### PR TITLE
Install EXTRA_PACKAGES to worker env

### PR DESCRIPTION
**Pull request by delgadom**
## Workflow

* [x] Closes issue #43 
* [ ] Notebook:worker pairing builds & passes local build, visual inspection & client.map tests
* [x] Passes travis tests
* [ ] Image deployed on `dev` branch (see [notebook](https://hub.docker.com/r/rhodium/notebook/tags/) and [worker](https://hub.docker.com/r/rhodium/worker/tags/) dev tags)
* [ ] Worker passes manual deployment test on compute.rhg
* [ ] Notebook+worker passes test-hub deployment
* [ ] Updates integrated into downstream images

## Summary

Currently `EXTRA_CONDA_PACKAGES` and `EXTRA_PIP_PACKAGES` arguments get installed to the root conda env. Moving them to `worker` on the worker images.

### TODO

* We should add something about extra_pip_packages to the standard test suite
* This is specifically affecting @bolliger32 so we should test this against the ClimateImpactLab/bolliger32-clawpack-image build

### CODE REVIEW

File: worker/prepare.sh [hunk 1]
```diff
@@ -11,12 +11,12 @@ fi
 
 if [[ "$EXTRA_CONDA_PACKAGES" ]]; then
     echo "EXTRA_CONDA_PACKAGES environment variable found.  Installing."
-    /opt/conda/bin/conda install --yes $EXTRA_CONDA_PACKAGES
+    /opt/conda/envs/worker/bin/conda install --yes $EXTRA_CONDA_PACKAGES
```
Comment (bolliger32) [inline worker/prepare.sh]: I don't know if there is anything else that being in a conda env does wouldn't another option just be to move the `source activate worker` line to the top of the script? And then just call `conda install` and `pip install`. Not sure if this approach is better?

Comment (delgadom) [reply worker/prepare.sh]: Habit. it has the same effect, and docker couldn't find source so this is how we install all the packages in the Dockerfile. We could use either here.

Comment (bolliger32) [reply worker/prepare.sh]: sounds good! In my own troubleshooting I figured out that docker can't find it b/c it uses /bin/sh by default, which doesn't support `source`. If we need it at any point, you can use the command `SHELL ["/bin/bash", "-c"]` prior to calling source. i.e.

```SHELL ["/bin/bash", "-c"]
RUN source activate worker && \
  pip install --src=/ -e git+https://github.com/ClimateImpactLab/clawpack.git#egg=clawpack --no-cache-dir
```

**Comment by delgadom** [comment]
Duude noice.

________________________________

**Pull request merged**